### PR TITLE
Feat/load css

### PR DESCRIPTION
**Pull request by tomwayson**
export a new function `loadFeatures(url)` that allows consumers to load the esri CSS by passing a URL (for now, more smarts to come later). Only loads the CSS once.

includes tests and API docs (README).

Propose merging this instead of #80 as the first step towards implementing some of the ideas discussed in that PR and in #6.

I think the next step would be adding a `css` option to `loadScript` that would take a string and simply call `loadCss(options.css)` if supplied.

Next we could allow the user to pass`css: true` and we could try to derive the URL based on the script's URL.

In all cases, if `options.css` is falsey, we don't try to load the styles.

What do you think @SimplyAutomationized?

### CODE REVIEW

**Comment by tomwayson** [comment]
Haven't tested this out in an actual app yet, so DNM yet.

**Comment by jwasilgeo** [comment]
Thanks for this. Looks promising so far. I will have the opportunity soon to do some testing with a couple front-end frameworks where it makes sense to ask esri-loader to handle the work of loading in the esri css.

File: src/esri-loader.ts [hunk 1]
```diff
@@ -72,6 +79,11 @@ export const utils = {
 export function getScript() {
   return document.querySelector('script[data-esri-loader]') as HTMLScriptElement;
 }
+// TODO: export this function?
```
Comment (jwasilgeo) [inline src/esri-loader.ts]: I suppose leave it un-exported for now?

**Comment by jwasilgeo** [comment]
@tomwayson here is a working example of using `loadCss` in a Riot app demonstration:
https://github.com/jwasilgeo/esri-riot-example/blob/master/index.html#L68

This is a neat test case because it calls `loadCss` as many times as there are unique `<esri-map-view>` Riot components mounted to the DOM.  esri-loader works as expected by only attempting to load the esri stylesheet once.

File: src/esri-loader.ts (hunk 1 above)
Comment (tomwayson) [reply src/esri-loader.ts]: I think for now, I want the `loadCss()` API to settle first

**Comment by tomwayson** [comment]
Thanks for trying this out @jwasilgeo!

You should PR a link to your riot example to the README! Very cool.

**Pull request merged**